beacon_node/beacon_chain/src/builder.rs

```diff
@@ -396,6 +399,12 @@ where
         self
     }
 
+    /// Sets the `graffiti` field.
+    pub fn graffiti(mut self, graffiti: &Graffiti) -> Self {
+        self.graffiti = graffiti.clone();
```

Comment:
I could change this to use the same code as [here](https://github.com/sigp/lighthouse/pull/1320/files#r447817125)

Reply:
I'm not sure where that link leads. This seems fine to me though. I would be tempted to give `Graffiti` as owned (not borrowed) since we always clone it, but that's very minor for 32 bytes haha.

Reply:
Lol link is indeed dead because I removed it, nvm :p
I had first used an owned version and then though Clippy would say "Hey you're passing something that's bigger than an address, consider just taking the address!" but actually Clippy doesn't complain :)

See https://github.com/sigp/lighthouse/pull/1320/commits/d849545631222310401803f6de2c546b5d47f153 :D